File: Facebook.NET/Contracts/GraphAPI.Pagination.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphApi
    {
        [DataContract]
        public sealed partial class Pagination
        {
            [DataMember(Name = "cursors")]
            public GraphApi.Pagination.Cursor Cursors { get; set; }

            [DataMember(Name = "next")]
            public string Previous { get; set; }

            [DataMember(Name = "next")]
            public string Next { get; set; }
        }
    }
}

File: Facebook.NET - VB/Contracts/GraphAPI.Pagination.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Partial Public NotInheritable Class Pagination
            <DataMember(Name:="cursors")>
            Public Property Cursors() As GraphApi.Pagination.Cursor
            <DataMember(Name:="next")>
            Public Property Previous() As String
            <DataMember(Name:="next")>
            Public Property [Next]() As String
        End Class
    End Class
End Namespace
